pyserini/search/lucene/tprob/_search_tprob.py

```diff
@@ -0,0 +1,231 @@
+#
+# Pyserini: Python interface to the Anserini IR toolkit built on Lucene
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+This module provides Pyserini's Python translation probability search
+interface on MS MARCO dataset. The main entry point is the
+``TranslationProbabilitySearcher`` class.
+"""
+from pyserini.pyclass import autoclass
+from pyserini.util import download_prebuilt_index
+from typing import Dict
+import math
+import struct
+import json
+from multiprocessing.pool import ThreadPool
+from pyserini.search import SimpleSearcher
+
+# Wrappers around Anserini classes
```

Comment:
I think these are already defined elsewhere?

Reply:
You mean the Anserini classes right? But some of them are defined in different files, it might be easier to just define them again in this file?

Reply:
okay - yes, I suppose it's the difference between re-defining and importing...

what you have is fine.